stable/insights-agent/templates/rbac-reporter/_job.yaml

```diff
@@ -0,0 +1,41 @@
+{{- define "rbacreporter.job.spec" -}}
+restartPolicy: Never
+serviceAccountName: {{ include "insights-agent.fullname" . }}
```

Comment:
I for one would like to see us start using different Service Accounts when one of the jobs needs special permissions like this. Not sure if I'm being paranoid but principal of least privilege is stuck in my head :)

Reply:
created separate service account